technologies/job/bash/metadata.yaml

```diff
@@ -5,36 +5,12 @@ label: Bash
 available: true
 icon: bash
 contexts:
-  - id: debian8-jessie
```

Comment:
if you delete this section, job on debian8 in saagie will become non-executable ?

Reply:
Since the SDK and catalog creation, debian-jessie is disabled https://github.com/saagie/technologies/pull/148/files so no jobs can exist with this version